utils/clone-llvm.sh

```diff
@@ -14,9 +14,6 @@
 
 export commithash=5e111eb275eee3bec1123b4b85606328017e5ee5
 
-git clone --depth 1 https://github.com/Xilinx/cmakeModules cmakeModules/cmakeModulesXilinx
```

Comment:
The root cause is that this should have used https://cmake.org/cmake/help/latest/module/FetchContent.html instead of this ad-hoc solution.